great_tables/_formats_vals.py

```diff
@@ -1,14 +1,16 @@
 from __future__ import annotations
 from ._gt_data import GTData
+from ._tbl_data import to_frame, SeriesLike
 from great_tables.gt import _get_column_of_values, GT
 from great_tables import GT
 from typing import List, Any, Union, Optional, TYPE_CHECKING
 
 if TYPE_CHECKING:
     from ._formats import DateStyle, TimeStyle
+    from ._tbl_data import SeriesLike
 
 
-def _make_one_col_table(vals: Union[Any, List[Any]]) -> GTData:
+def _make_one_col_table(vals: Union[Any, List[Any], SeriesLike]) -> GT:
```

Comment:
One challenge here is that, if a user passes a list, then we have to use a pandas Series on it.

By adding SeriesLike, it lets users pass their own series (pandas or polars), so we don't have to depend on a specific one. 

See #266